server/src/com/cloud/network/router/VpcVirtualNetworkApplianceManagerImpl.java

```diff
@@ -455,16 +471,26 @@ protected void finalizeNetworkRulesForNetwork(final Commands cmds, final DomainR
 
         super.finalizeNetworkRulesForNetwork(cmds, domainRouterVO, provider, guestNetworkId);
 
+        final VpcVO vpc = _vpcDao.findById(domainRouterVO.getVpcId());
         if (domainRouterVO.getVpcId() != null) {
-
             if (domainRouterVO.getState() == State.Starting || domainRouterVO.getState() == State.Running) {
                 if (_networkModel.isProviderSupportServiceInNetwork(guestNetworkId, Service.NetworkACL, Provider.VPCVirtualRouter)) {
                     final List<NetworkACLItemVO> networkACLs = _networkACLMgr.listNetworkACLItems(guestNetworkId);
                     if (networkACLs != null && !networkACLs.isEmpty()) {
-                        s_logger.debug("Found " + networkACLs.size() + " network ACLs to apply as a part of VPC VR " + domainRouterVO + " start for guest network id=" + guestNetworkId);
+                        s_logger.debug(
+                                "Found " + networkACLs.size() + " network ACLs to apply as a part of VPC VR " + domainRouterVO + " start for guest network id=" + guestNetworkId);
```

Comment:
While I recognize this change is due a formatting change, would it be possible to wrap this log statement in an ``s_logger.isDebugEnabled()`` check to avoid unnecessary string concatenation when ``DEBUG`` logging is not enabled?